src/common/routes.js

```diff
@@ -109,32 +109,41 @@ export const conferenceEventsHandler = async (req, res) => {
       .conferences(ConferenceSid)
       .participants.list();
 
-    const isParticipantLeave = StatusCallbackEvent === 'participant-leave';
-
     // Send the conference sid and the other participants' callSids to any client leg.
     // The sids will be used to update the participant resource such as putting it on hold.
-    participants.forEach((currentParticipant) => {
+    participants.forEach(async (currentParticipant) => {
```

Comment:
Is there any UI logic that happens around this? Should we map each async function to a Promise and await `Promise.all`?

Reply:
@mhuynh5757 Yes there is UI logic that happens, but the UI logic happens with each `UserDefinedMessage`. Even if we await all these messages in `Promise.all`, the `UserDefinedMessage(s)` still send as individual messages. The API does not return the messages. Or do you think we should handle these messages differently?

Reply:
As in, the UI changes when receiving the messages? Ah I see, this is the `routes.js` file so this is on the server side. I think it's good as-is then.